### PR TITLE
[zh-cn]: init the translation of HTTP status 102

### PR DESCRIPTION
**Pull request by Hsiao-Feng**
### Description

Translate from [English version](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/102).

### Motivation

New translation.

### Additional details

Link: https://developer.mozilla.org/zh-CN/docs/Web/HTTP/Status/102

### CODE REVIEW

File: files/zh-cn/web/http/status/102/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+slug: Web/HTTP/Status/102
+page-type: http-status-code
+spec-urls: https://www.rfc-editor.org/rfc/rfc2518.html#section-10.1
+l10n:
+  sourceCommit: 592f6ec42e54981b6573b58ec0343c9aa8cbbda8
+---
+
+{{HTTPSidebar}}{{deprecated_header}}
+
+
```
[the remark this answers is not in the document]
Comment (Hsiao-Feng) [reply files/zh-cn/web/http/status/102/index.md:10]: I think these shouldn't be removed.

File: files/zh-cn/web/http/status/102/index.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: 102 Processing
+slug: Web/HTTP/Status/102
+l10n:
+  sourceCommit: 592f6ec42e54981b6573b58ec0343c9aa8cbbda8
+---
+
+{{HTTPSidebar}}{{deprecated_header}}
+
+
```
Comment (yin1999) [inline files/zh-cn/web/http/status/102/index.md:10]:
```suggestion
```

File: files/zh-cn/web/http/status/102/index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+在 HTTP 协议中，**`102 Processing`** 状态码向客户端表示已收到完整请求，并且服务器正在处理该请求。
+
+仅当服务器预计请求需要很长时间时会发送此状态码，以告知客户端请求尚未终止。
+
+> **注意：** 此状态码已弃用，不应再发送。客户可能仍然接受，但会忽略。
```
Comment (yin1999) [inline files/zh-cn/web/http/status/102/index.md:15]: I'm not really sure does this translation could express its meaning.

```suggestion
> **备注：** 此状态码已被弃用，不应再发送。客户端可能仍然接受并简单地忽略该状态码。
```

File: files/zh-cn/web/http/status/102/index.md (hunk 2 above)
Comment (Hsiao-Feng) [reply files/zh-cn/web/http/status/102/index.md:10]: Don't need these templates?

**Comment by yin1999** [comment]
Please read the guidelines: https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md

File: files/zh-cn/web/http/status/102/index.md (hunk 2 above)
Comment (yin1999) [reply files/zh-cn/web/http/status/102/index.md:10]: Only one blank line is needed between paragraphs

File: files/zh-cn/web/http/status/102/index.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+102 Processing
+```
+
+## 规范
+
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
```
Comment (yin1999) [inline files/zh-cn/web/http/status/102/index.md:28]: Remove bcd

```suggestion
```

**Review by yin1999 — APPROVED** [review]
Thanks

**Comment by Hsiao-Feng** [comment]
> Please read the guidelines: https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md

Thanks for your help :D

**Pull request merged**